pkg/ocm/output/nodepools_test.go

```diff
@@ -16,17 +16,31 @@ var _ = Describe("Create node drain grace period builder validations", func() {
 			output := PrintNodeDrainGracePeriod(period)
 			Expect(output).To(Equal(expectedOutput))
 		},
-		Entry(nil,
+		Entry("Should return empty string", nil,
```

Comment:
The test file never runs without the output_test.go. It was missing the description.